samples/to_dataframe/main_test.py

```diff
@@ -98,48 +96,46 @@ def test_query_to_dataframe(capsys, clients):
 
 
 def test_session_to_dataframe(capsys, clients):
-    from google.cloud import bigquery_storage_v1beta1
+    from google.cloud.bigquery.storage import types
 
     bqclient, bqstorageclient = clients
     your_project_id = bqclient.project
 
     # [START bigquerystorage_pandas_tutorial_all]
     # [START bigquerystorage_pandas_tutorial_read_session]
-    table = bigquery_storage_v1beta1.types.TableReference()
-    table.project_id = "bigquery-public-data"
-    table.dataset_id = "new_york_trees"
-    table.table_id = "tree_species"
+    project_id = "bigquery-public-data"
+    dataset_id = "new_york_trees"
+    table_id = "tree_species"
+    table = f"projects/{project_id}/datasets/{dataset_id}/tables/{table_id}"
 
     # Select columns to read with read options. If no read options are
     # specified, the whole table is read.
-    read_options = bigquery_storage_v1beta1.types.TableReadOptions()
-    read_options.selected_fields.append("species_common_name")
-    read_options.selected_fields.append("fall_color")
+    read_options = types.ReadSession.TableReadOptions(
+        selected_fields=["species_common_name", "fall_color"]
+    )
 
     parent = "projects/{}".format(your_project_id)
-    session = bqstorageclient.create_read_session(
-        table,
-        parent,
-        read_options=read_options,
+
+    requested_session = types.ReadSession(
+        table=table,
         # This API can also deliver data serialized in Apache Avro format.
         # This example leverages Apache Arrow.
-        format_=bigquery_storage_v1beta1.enums.DataFormat.ARROW,
-        # We use a LIQUID strategy in this example because we only read from a
-        # single stream. Consider BALANCED if you're consuming multiple streams
-        # concurrently and want more consistent stream sizes.
-        sharding_strategy=(bigquery_storage_v1beta1.enums.ShardingStrategy.LIQUID),
```

Comment:
Where did the `ShardingStrategy` go, I could not find it among generated message types?

Reply:
When BQ Storage API launched to GA it switched to using BALANCED and dropped the LIQUID option.

Reply:
OK great, then the modified sample should be fine as-is.